src/Module/Api/Friendica/Photo/Create.php

```diff
@@ -55,7 +55,7 @@ protected function post(array $request = [])
 		$type = $this->getRequestValue($this->parameters, 'extension', 'json');
 
 		// input params
-		$desc      = $this->getRequestValue($request, 'desc');
+		$desc      = $this->getRequestValue($request, 'desc') ?? '';
```

Comment:
The function does handle default values, so it should look this way:
 `$desc = $this->getRequestValue($request, 'desc','');`

Reply:
Ah, right, I forgot about that.